go.mod

```diff
@@ -145,7 +145,7 @@ replace (
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-	github.com/tendermint/tendermint => github.com/sei-protocol/sei-tendermint v0.1.181
+	github.com/tendermint/tendermint => ../sei-tendermint
```

Comment:
will update once https://github.com/sei-protocol/sei-tendermint/pull/125 is merged